lib/plugins/aws/package/compile/events/cloudFront/index.js

```diff
@@ -206,16 +206,23 @@ class AwsCompileCloudFrontEvents {
               Object.assign(behavior, event.cloudFront.behavior);
             }
 
+            const lambdaFunctionAssociation = {
+              EventType: event.cloudFront.eventType,
+              LambdaFunctionARN: {
+                Ref: lambdaVersionLogicalId,
+              },
+            };
+
+            if (
+              event.cloudFront.includeBody !== null &&
+              event.cloudFront.includeBody !== undefined
+            ) {
```

Comment:
Let's write it simply as `if (event.cloudFront.includeBody != null)`

Reply:
sure